app/models/queue_item.rb

```diff
@@ -0,0 +1,19 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+  validates_presence_of :video_id, :user_id
```

Comment:
I am not sure why are you validating this. Is user setting these directly via form? Whatever your app sets does not have to be validated as you should have tests for that.